packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/CameraPlugin.m

```diff
@@ -118,26 +118,52 @@ - (void)availableCamerasWithCompletion:
     NSArray<AVCaptureDevice *> *devices = discoverySession.devices;
     NSMutableArray<FCPPlatformCameraDescription *> *reply =
         [[NSMutableArray alloc] initWithCapacity:devices.count];
+
     for (AVCaptureDevice *device in devices) {
       FCPPlatformCameraLensDirection lensFacing;
-      switch (device.position) {
-        case AVCaptureDevicePositionBack:
-          lensFacing = FCPPlatformCameraLensDirectionBack;
-          break;
-        case AVCaptureDevicePositionFront:
-          lensFacing = FCPPlatformCameraLensDirectionFront;
-          break;
-        case AVCaptureDevicePositionUnspecified:
-          lensFacing = FCPPlatformCameraLensDirectionExternal;
-          break;
-      }
+      FCPPlatformCameraLensType lensType;
+      getLensDirectionAndType(device, &lensFacing, &lensType);
+
       [reply addObject:[FCPPlatformCameraDescription makeWithName:device.uniqueID
-                                                    lensDirection:lensFacing]];
+                                                    lensDirection:lensFacing
+                                                         lensType:lensType]];
     }
     completion(reply, nil);
   });
 }
 
+static void getLensDirectionAndType(AVCaptureDevice *device,
```

Comment:
Nit: this can be just a global function outside of this class. Also can you follow naming convention `FCPGetLensDirectionAndType`